### PR TITLE
style(settings): code cleanup

### PR DESCRIPTION
**Pull request by andrewazores**
General code style cleanup and improvements. Mostly cosmetic, with some improvements to Subscription handling

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @andrewazores for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by joshuawilson** [comment]
Would you be willing to apply these changes to the `upgrade` branch. I think some of them would be lost in a merge or it will just be a hard merge as the imports have changes.

File: src/app/profile/settings/connected-accounts/connected-accounts.component.ts [hunk 1]
```diff
@@ -26,30 +25,32 @@ export class ConnectedAccountsComponent implements OnDestroy, OnInit {
...
 
-  constructor(private contexts: Contexts,
-    private auth: AuthenticationService,
-    private userService: UserService,
-    private providerService: ProviderService) {
-    this.subscriptions.push(auth.gitHubToken.subscribe(token => {
+  private readonly subscriptions: Subscription[] = [];
+
+  constructor(
+    contexts: Contexts,
```
Comment (aptmac) [inline src/app/profile/settings/connected-accounts/connected-accounts.component.ts]: out of curiosity, why the change away from private?

File: src/app/profile/settings/connected-accounts/connected-accounts.spec.ts [hunk 2]
```diff
@@ -2,8 +2,8 @@ import { Component } from '@angular/core';
 import { Contexts } from 'ngx-fabric8-wit';
 import { AuthenticationService } from 'ngx-login-client';
 import { UserService } from 'ngx-login-client';
-import { empty as observableEmpty, Observable,  of ,  throwError as observableThrowError } from 'rxjs';
-import { initContext, TestContext } from 'testing/test-context';
+import { empty as observableEmpty, of , throwError } from 'rxjs';
```
Comment (aptmac) [inline src/app/profile/settings/connected-accounts/connected-accounts.spec.ts]: similar to the changes in the deployments PR, should `empty as observableEmpty` simply become `empty`?

Comment (joshuawilson) [reply src/app/profile/settings/connected-accounts/connected-accounts.spec.ts]: If you do make that switch use the upper case EMPTY.

File: src/app/profile/settings/connected-accounts/connected-accounts.component.ts (hunk 1 above)
Comment (andrewazores) [reply src/app/profile/settings/connected-accounts/connected-accounts.component.ts]: The `private` keyword there makes the parameters become private members of the class, but these parameters were only referenced directly within the constructor body and nowehere else in the class, so there is no need to make them class members.
